install.yml

```diff
@@ -76,7 +76,8 @@
         state: present
       with_items:
         - 'gem "blacklight", "{{ blacklight_version }}"'
-        - 'gem "blacklight-spotlight", "{{ spotlight_version }}"'
+        - "gem 'blacklight-spotlight', github: 'projectblacklight/spotlight'"
```

Comment:
This uses the latest in github, regardless of the spotlight_version defined in vars.yml.